wren-ui/src/apollo/server/services/askingService.ts

```diff
@@ -322,6 +330,11 @@ export class AskingService implements IAskingService {
    * 3. put the task into background tracker
    */
   public async createThread(input: AskingDetailTaskInput): Promise<Thread> {
+    // if input contains a viewId, simply create a thread from saved properties of the view
+    if (input.viewId) {
+      return this.createThreadFromView(input.viewId);
+    }
+
     // 1. create a task on AI service to generate the detail
```

Comment:
I think we can wrap L338-363 to a method called "createThreadFromLLM"  and move the comment around L330